### DISCUSSION TITLE
Dropdown bug?

### DISCUSSION THREAD

**Discussion by eu16** [quasarframework/quasar#9856] [General - Components / Directives / etc]
Does anyone know why that Quasar's dropdown is pushed to one side at full screen? But not when its on mobile mode?

<img width="1032" alt="Screenshot 2021-06-29 at 3 33 05 PM" src="https://user-images.githubusercontent.com/85152661/123756580-a57abd80-d8ef-11eb-9315-2b977af8dc8f.png">

Sample Code - using VueJS

<template>
  <div class="container">
    <div class="bg-white" style="box-shadow: 0 10px 6px -6px #777">
      <br />
      <q-toolbar class="q-my-md row">
        <q-btn flat v-on:click="lastPage()"
          ><q-icon class="q-mt-sm" size="2rem" name="fas fa-angle-left"
        /></q-btn>
        <h3 class="title">{{ $t("settings.welcome") }}</h3>
      </q-toolbar>
      <br />
    </div>
    <br />

    <div class="settingsForm">
      <br />
      <div
        class="text-h5 q-mt-md text-weight-bold text-center"
        style="color: rgb(4, 155, 219)"
      >
        <i class="fas fa-user"></i>
      </div>
      <q-card class="my-card" inline style="border-radius: 10px">
        <q-card-section class="text-h6">
          
          {{ $t("settings.welcome") }}
        </q-card-section>

        <q-card-section class="">
          Please Choose a Language / 請選擇語言:
        </q-card-section>

        <q-card-section>
          <q-btn-dropdown class="full-width" align="left" label="--choose--">
            <q-list>
              <q-item clickable v-close-popup @click="onItemClick">
                <q-item-section>
                  <q-item-label>English</q-item-label>
                </q-item-section>
              </q-item>

              <q-item clickable v-close-popup @click="onItemClick">
                <q-item-section>
                  <q-item-label>简体中文</q-item-label>
                </q-item-section>
              </q-item>

              <q-item clickable v-close-popup @click="onItemClick">
                <q-item-section>
                  <q-item-label>繁體中文</q-item-label>
                </q-item-section>
              </q-item>
            </q-list>
          </q-btn-dropdown>
        </q-card-section>

        <q-card-section class="">
          <q-btn
            no-caps
            color="light-blue-7"
            align="center"
            class="full-width my-btn"
            label="Start the Chat"
          ></q-btn>
        </q-card-section>
      </q-card>
    </div>
  </div>
</template>

<script>
export default {
  methods: {
    lastPage: function () {
      this.$router.back();
    },
    setup() {
      return {
        onItemClick() {
          // console.log('Clicked on an Item')
        },
      };
    },
  },
};
</script>

**Comment by metalsadman** [comment]
please provide a reproduction codepen.
